composer.json

```
@@ -25,7 +25,7 @@
         }
     ],
     "require": {
-        "php": ">=7.1.0,<7.5.0",
+        "php": ">=7.1.0,<7.5.0|^8.0",
```

This should be

```suggestion
        "php": ">=7.1.0,<7.5.0|>=8.0.0,<8.1.0",
```

Since PHP 8.1 is not released yet, there is no possible way we can know whether there will be BC breaks we cannot account for yet :+1: